### ISSUE TITLE
The options from app's shortcut don't work on Focus Nightly

### ISSUE THREAD

**Issue by lobontiumira** [mozilla-mobile/focus-android#5851]
## Steps to reproduce
1. Install the 11/11 Focus Nightly 95.0 (build 353151018 GV 96.0a1-20211110092453).
2. Long-tap on the Focus icon from the device's homescreen.
3. Tap on the "Erase & open" option.

### Expected behavior
The app's history is erased, the app is opened.

### Actual behavior
Nothing happens. The app isn't opened.

```
2021-11-11 12:33:19.571 914-2390/? W/ActivityTaskManager: Permission Denial: starting Intent { act=erase flg=0x1000c000 cmp=org.mozilla.focus/.activity.EraseAndOpenShortcutActivity bnds=[63,290][641,437] (has extras) } from null (pid=-1, uid=10435) not exported from uid 10431
2021-11-11 12:33:19.578 1807-1807/? E/BaseDraggingActivity: Unable to launch. tag=WorkspaceItemInfo(id=-1 type=DEEPSHORTCUT container=-1 screen=-1 cell(-1,-1) span(1,1) minSpan(1,1) rank=1 user=UserHandle{0} title=Erase & open) intent=Intent { act=android.intent.action.MAIN cat=[com.android.launcher3.DEEP_SHORTCUT] flg=0x10200000 pkg=org.mozilla.focus.nightly cmp=org.mozilla.focus.nightly/org.mozilla.focus.activity.MainActivity bnds=[63,290][641,437] (has extras) }
    android.content.ActivityNotFoundException: Shortcut could not be started
        at android.content.pm.LauncherApps.startShortcut(LauncherApps.java:1144)
        at android.content.pm.LauncherApps.startShortcut(LauncherApps.java:1109)
        at com.android.launcher3.shortcuts.DeepShortcutManager.startShortcut(SourceFile:1)
        at com.android.launcher3.BaseDraggingActivity.startShortcutIntentSafely(SourceFile:10)
        at com.android.launcher3.BaseDraggingActivity.startActivitySafely(SourceFile:11)
        at com.android.launcher3.Launcher.startActivitySafely(SourceFile:6)
        at com.android.launcher3.touch.ItemClickHandler.startAppShortcutOrInfoActivity(SourceFile:15)
        at com.android.launcher3.touch.ItemClickHandler.onClickAppShortcut(SourceFile:13)
        at com.android.launcher3.touch.ItemClickHandler.a(SourceFile:12)
        at c.a.a.p.c.onClick(Unknown Source:2)
        at android.view.View.performClick(View.java:7140)
        at com.android.launcher3.shortcuts.DeepShortcutTextView.performClick(SourceFile:9)
        at android.view.View.performClickInternal(View.java:7117)
        at android.view.View.access$3500(View.java:801)
        at android.view.View$PerformClick.run(View.java:27351)
        at android.os.Handler.handleCallback(Handler.java:883)
        at android.os.Handler.dispatchMessage(Handler.java:100)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)

```
### Device information
* Android device: Google Pixel (Android 10), Samsung Galaxy Note 8 (Android 9)
* Focus version: Focus Nightly 95.0 (build 353151018 GV 96.0a1-20211110092453)

https://user-images.githubusercontent.com/40694158/141284570-0affd25c-7f5e-4864-bf18-c5b557e2fd09.mp4

**Comment by mcarare** [comment]
@lobontiumira Does this happen on the latest main? I tested on an Android 10 Nokia and an Android 9 Huawei P10, and it works as expected.

**Comment by lobontiumira** [comment]
@mcarare it works on the latest main from debug.

**Comment by mcarare** [comment]
Thanks for the confirmation,  added a fix for Nightly.

**Comment by lobontiumira** [comment]
Verified as fixed on the latest Focus Nightly 95.0 (build 353160913 GV 96.0a1-20211111045525) with Google Pixel (Android 10).
